pkg/storage/access_token_file_test.go

```diff
@@ -0,0 +1,149 @@
+package storage
+
+import (
+	storageConfig "github.com/cyberark/conjur-authn-k8s-client/pkg/storage/config"
+	. "github.com/smartystreets/goconvey/convey"
+	"os"
+	"reflect"
+	"testing"
+)
+
+type ProxyHandlerTokenFile struct {
+	AccessToken AccessTokenHandler
+}
+
+func TestAccessTokenFile(t *testing.T) {
+	var config storageConfig.Config
+	config.StoreType = storageConfig.K8S
+	config.TokenFilePath = "/tmp/accessTokenFile1"
+	var tokenInFile, _ = NewAccessTokenFile(config)
+
+	Convey("Read", t, func() {
+		Convey("Returns true if Data output as expected", func() {
+			dataActual := []byte{'t', 'e', 's', 't'}
+			err := tokenInFile.Write(dataActual)
+			So(err, ShouldEqual, nil)
+
+			dataExpected, err := tokenInFile.Read()
+			eq := reflect.DeepEqual(dataActual, dataExpected)
```

Comment:
when would dataActual and dataExpected ever be different? You just wrote dataActual and then immediately read it so why is this `So(eq, ShouldEqual, true)` a good indicator that our code works?